prometheus-external-monitor/templates/endpoint.yaml

```diff
@@ -0,0 +1,20 @@
+{{- $root := .}}
+{{- range .Values.endpoints }}
+---
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: {{ template "prometheus-external-monitor.fullname" $root }}-{{ .name }}
+  labels:
+    app: {{ template "prometheus-external-monitor.name" $root }}-{{ .name }}
+    chart: {{ template "prometheus-external-monitor.chart" $root }}
+    release: {{ $root.Release.Name }}
+    heritage: {{ $root.Release.Service }}
+subsets:
+- addresses: 
+  - ip: {{ .ip }}
+  ports: 
+  - name: metrics
+    port: {{ .port | default "9100" }}
+    protocol: TCP
+{{- end }}
```

Comment:
Please do not commit files without and ending newline

Reply:
Need to put something on my editor...